README.md

```diff
@@ -146,6 +146,7 @@ Table of Contents
 
   * https://bitnami.com/ - Deploy prepared apps on IaaS. Management of 1 AWS micro instance free
   * http://www.visualops.io/ - 3,600 instance hours per month free
+  * http://devopsboard.com/ - Manage software and hardware issues aggregated from monitoring tools on single Kanban board
```

Comment:
All I see on their pricing page regarding it is:

```Submit the credentials on the left and get your instance ready in couple of seconds. We give you 14-days trial period to use DevOps board absolutely free.```

which suggests its a trial only